schema/scope.go

```diff
@@ -149,7 +149,7 @@ func (s *ScopeSchema) RootObject() *ObjectSchema {
 	}
 	if rootObject.ID() != s.RootValue {
 		panic(fmt.Sprintf(
-			"root object's ID %q doesn't match its map key %q; please fix the input",
+			"root object's ID %q doesn't match its map key %q; please fix the schema definition",
 			rootObject.ID(), s.RootValue,
 		))
```

Comment:
This is better, but I really think it needs the phrase "user error":  without that, the user can't tell whether that instruction is for them or for the Arca developers.

Reply:
We just need to be careful. Because this is the SDK. We shouldn't make assumptions on who wrote the schema here.
In the follow-up task, that will make us better able to use context to provide useful errors. We will need to report the error up the chain to the caller, which will be the plugin or engine. The caller can then provide the context.

Reply:
Once the followup task is complete, we should never see the message, so hopefully it won't matter for very long what we say here.  But, if we don't know who wrote the schema, we probably shouldn't be telling the recipient of this message to fix it.